src/release/breaking-changes/window-singleton.md

```diff
@@ -0,0 +1,144 @@
+---
+title: The window singleton is deprecated
+description: In preparation for supporting multiple views and multiple windows the window singleton has been deprecated.
+---
+
+## Summary
+
+Originally, Flutter assummed that an application would only consist of a single
+view (the `window`) into which content can be drawn. In preparation for
+supporting multiple views and multiples windows, this single view assumtion and
+the APIs that exposed it have been deprecated. Applications and libraries that
+relied on these APIs need to migrate to new multi-view compatible APIs as
+outlined in this migration guide.
+
+## Context
+
+In a multi-view world APIs that just assume a single window do not make sense
+anymore. Instead, applications and libraries must chose the specific
+view they want to operate on. 
+
+## Description of change
+
+The APIs that have been deprecated as part of this change are:
+
+* the global `window` property exposed by `dart:ui`
+* the `window` property on the `BaseBinding` class, 
+  which is usually accessed via
+  * `GestureBinding.instance.window`
+  * `SchedulerBinding.instance.window`
+  * `ServicesBinding.instance.window`
+  * `PaintingBinding.instance.window`
+  * `SemanticsBinding.instance.window`
+  * `RendererBinding.instance.window`
+  * `WidgetsBinding.instance.window`
+  * `WidgetTester.binding.window`
+* the `SingletonFlutterView` class from `dart:ui`
+* TODO: mention any test related classes/properties (e.g. TestWindow)
+
+The following options exist to migrate application and library code that relies
+on these deprecated APIs:
+
+If a `BuildContext` is available, consider looking up the current `FlutterView`
```

Comment:
Is second person acceptable in migration docs? If so, consider something like:

```suggestion
If a `BuildContext` is available, you can use `View.of` to get the current `FlutterView`.
```

Reply:
I think migration guides should be in passive voice like our API docs?